pylint/pyreverse/main.py

```diff
@@ -111,7 +112,7 @@
         "module-names",
         dict(
             short="m",
-            default=None,
+            default="no",
```

Comment:
Most common default value I found across the code base for ``type="yn"`` options was using booleans directly, but looking at the the ``_yn_transformer`` this should be fine as well.

Reply:
That's actually more performant. Going to change to booleans!

Reply:
Upon a closer look, we can't change that, it actually alters the behavior (see above: the ``pyreverse`` code explicitly checks for ``is None`` in some cases).